packages/trader/src/AppV2/Components/SymbolIconsMapper/symbol-icons-mapper.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import {
+    MarketCommodityGoldusdIcon,
+    MarketCommodityPalladiumusdIcon,
+    MarketCommodityPlatinumusdIcon,
+    MarketCommoditySilverusdIcon,
+    MarketCryptocurrencyBtcusdIcon,
+    MarketCryptocurrencyEthusdIcon,
+    MarketDerivedAudBasketIcon,
+    MarketDerivedBearIcon,
+    MarketDerivedBoom1000Icon,
+    MarketDerivedBoom300Icon,
+    MarketDerivedBoom500Icon,
+    MarketDerivedBullIcon,
+    MarketDerivedCrash1000Icon,
+    MarketDerivedCrash300Icon,
+    MarketDerivedCrash500Icon,
+    MarketDerivedEurBasketIcon,
+    MarketDerivedGbpBasketIcon,
+    MarketDerivedGoldBasketIcon,
+    MarketDerivedJump10Icon,
+    MarketDerivedJump100Icon,
+    MarketDerivedJump25Icon,
+    MarketDerivedJump50Icon,
+    MarketDerivedJump75Icon,
+    MarketDerivedStepIcon,
+    MarketDerivedUsdBasketIcon,
+    MarketDerivedVolatility1001sIcon,
+    MarketDerivedVolatility100Icon,
+    MarketDerivedVolatility101sIcon,
+    MarketDerivedVolatility10Icon,
+    MarketDerivedVolatility1501sIcon,
+    MarketDerivedVolatility251sIcon,
+    MarketDerivedVolatility2501sIcon,
+    MarketDerivedVolatility25Icon,
+    MarketDerivedVolatility501sIcon,
+    MarketDerivedVolatility50Icon,
+    MarketDerivedVolatility751sIcon,
+    MarketDerivedVolatility75Icon,
+    MarketForexAudcadIcon,
+    MarketForexAudchfIcon,
+    MarketForexAudjpyIcon,
+    MarketForexAudnzdIcon,
+    MarketForexAudusdIcon,
+    MarketForexEuraudIcon,
+    MarketForexEurcadIcon,
+    MarketForexEurchfIcon,
+    MarketForexEurgbpIcon,
+    MarketForexEurjpyIcon,
+    MarketForexEurnzdIcon,
+    MarketForexEurusdIcon,
+    MarketForexGbpaudIcon,
+    MarketForexGbpcadIcon,
+    MarketForexGbpchfIcon,
+    MarketForexGbpjpyIcon,
+    MarketForexGbpnokIcon,
+    MarketForexGbpnzdIcon,
+    MarketForexGbpusdIcon,
+    MarketForexNzdjpnIcon,
+    MarketForexNzdusdIcon,
+    MarketForexUsdcadIcon,
+    MarketForexUsdchfIcon,
+    MarketForexUsdjpyIcon,
+    MarketForexUsdmxnIcon,
+    MarketForexUsdnokIcon,
+    MarketForexUsdplnIcon,
+    MarketForexUsdsekIcon,
+    MarketIndicesAustralia200Icon,
+    MarketIndicesEuro50Icon,
+    MarketIndicesFrance40Icon,
+    MarketIndicesGerman40Icon,
+    MarketIndicesHongKong50Icon,
+    MarketIndicesJapan225Icon,
+    MarketIndicesNetherlands25Icon,
+    MarketIndicesSwiss20Icon,
+    MarketIndicesUk100Icon,
+    MarketIndicesUs500Icon,
+    MarketIndicesUsTech100Icon,
+    MarketIndicesWallStreet30Icon,
+} from '@deriv/quill-icons';
+
+type TSymbolIconsMapper = {
+    symbol: string;
+};
+
+const SymbolIconsMapper = ({ symbol }: TSymbolIconsMapper) => {
+    const iconSize = 'md';
+
+    const iconMap = {
```

Comment:
May it be reused somehow in future (for getting correct icon)? Maybe in that case we should move it to utils

Reply:
um ya i can move the iconMap to utils